integration-tests/deployment/memory/job_client.go

```diff
@@ -156,6 +158,10 @@ func (j JobClient) DeleteJob(ctx context.Context, in *jobv1.DeleteJobRequest, op
 	panic("implement me")
 }
 
+func (j JobClient) NodeClients() []*client.ChainlinkK8sClient {
```

Comment:
Don't think we need to have this method here. This JobClient should closely match with [JD interface](https://github.com/smartcontractkit/chainlink/tree/develop/integration-tests/deployment/jd). We don't really need client.ChainlinkK8sClient, we will mainly work with node's [web/sdk graphql client](https://github.com/smartcontractkit/chainlink/tree/develop/integration-tests/web/sdk)

Reply:
@AnieeG if we don't add it, how will we execute interaction with nodes that are not supported by JD?